arlo-client/docs/api_sequence.md

```diff
@@ -764,6 +731,370 @@ happened on the first round, would look like this:
   ],
   name: 'contest name',
   randomSeed: '123456789',
-  riskLimit: '1',
+  riskLimit: 1,
+  online: true,
+}
+```
+
+### Form Three flow (ONLINE)
+
+If the audit has `online: true` the data is entered by the audit board members
+in the Data Entry Flow and there is no form shown here. Instead, a list of the
+ballots is fetched and a progress bar is displayed indicating how many of the
+ballots have been audited.
+
+- `GET /election/<electionId>/jurisdiction/<jurisdictionId>/round/<roundId>/ballot-list`
+
+- `src/components/AuditForms/CalculateRiskMeasurement.tsx` >
+  `CalculateRiskMeasurement` > `getBallots(round)` > `api()`
+
+  ```
+  {
+    ballots: [
+      {
+        comment: null,
+        position: 1,
+        status: null, // status is null!
+        timesSamples: 4,
+        vote: null,
+        batch: {
+          id: 'batch-1',
+          name: '1',
+          tabulator: null,
+        },
+        auditBoard: {
+          name: 'audit-board-1',
+          name: 'Audit Board #1',
+        },
+      },
+      ...
+    ]
+  }
+  ```
+
+Once all of the ballots returned have `status: 'AUDITED'` the progress bar will
+show completion and allow the round to be submitted for risk calculation.
+
+- `POST /election/{electionId}/jurisdiction/<jurisdiction_id>/<round_num>/results`
+
+- `src/components/AuditForms/CalculateRiskMeasurement.tsx` >
+  `CalculateRiskMeasurement` > `calculateRiskMeasurement()` > `api()`
+
+```
+{
+	"contests": [
+		{
+			id: "contest-1",
+   			results: {
+				"candidate-1": 0, // The backend doesn't care about these values, since they are supplied separately
```

Comment:
That seems odd. Why are we requiring and documenting them if they're ignored? Or is this the response?

Reply:
@eventualbuddha This is what gets sent and what prompts the calculation for the next round. It definitely is odd, yes. It's leftover from before we separated the offline and online data entry flows, and should be updated to something that makes more sense. I didn't even realize this was still done this way until I was walking through and documenting, haha.

Reply:
https://github.com/votingworks/arlo/issues/271